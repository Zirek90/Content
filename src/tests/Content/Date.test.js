import React from "react";
import renderer from "react-test-renderer";
import DateComponent from "../../components/Content/Date";

describe("DateComponent", () => {
  it("renders correctly", () => {
    const props = "2020-09-06T22:00:00Z";

    const tree = renderer.create(<DateComponent date={props} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
