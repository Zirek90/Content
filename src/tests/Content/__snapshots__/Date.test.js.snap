// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DateComponent renders correctly 1`] = `
<p
  className="sc-gtssRu kZfLwQ"
>
  Sun, 06 Sep 2020 22:00:00 GMT
</p>
`;
